shared/test/test_rack_attack.rb

```diff
@@ -143,6 +144,8 @@ def test_table_read_limits_with_exponential_backoff_and_logging
     # Now test that a dynamic update to the rate limit is enforced correctly.
     Timecop.freeze rounded_time_now + 60
     _test_table_dynamic_rate_limit_enforced
+    Timecop.freeze rounded_time_now + 60
```

Comment:
Good point. This works because Rack Attack checks the blacklist before checking throttles.